src/pages/instances/InstanceDetailHeader.tsx

```diff
@@ -58,6 +57,7 @@ const InstanceDetailHeader: FC<Props> = ({
           const instanceLink = instanceLinkFromName({
             instanceName: values.name,
             project,
+            instanceType: instance?.type || "container",
```

Comment:
```suggestion
            instanceType: instance?.type || "container",
            project,
```
Better keep this ordered?